go.mod

```diff
@@ -8,5 +8,6 @@ require (
 	github.com/ipfs/go-log v1.0.4
 	github.com/rs/cors v1.7.0
 	github.com/texttheater/golang-levenshtein v0.0.0-20180516184445-d188e65d659e
-	golang.org/x/crypto v0.0.0-20200115085410-6d4e4cb37c7d
+	github.com/tron-us/go-common/v2 v2.1.8
```

Comment:
Need to bump to v2.1.9?

Reply:
not really. latest release of go-common is v2.1.8

Reply:
i thought there is an update on go common master that resolved the warning issue, thats why we delayed the update?